packages/webapp/src/assets/mixin.scss

```diff
@@ -21,3 +21,15 @@
 @mixin smallShadow2() {
   box-shadow: 0px 2px 4px rgba(102, 115, 138, 0.3);
 }
+
+@mixin mobile-breakpoint {
```

Comment:
This is a great mixin -- super useful. Just wondering if there was further discussion with Loic on the breakpoints we want to use?  I see 415, and 1024 on figma. But I might prefer MUI's defaults https://mui.com/material-ui/customization/breakpoints/ which are closer to this number you have here. (But I overrided these for my PR).

Reply:
Ohhh yes, I should have added a comment in here! I talked about this with Loïc in my 1:1 today and the 415px breakpoint caused the Finances screen to overflow when it was between 415 and 600px, so he agreed to make that breakpoint a little wider. I see that 600px actually matches the MUI breakpoints so we could use those! I could rename this mixins to be something like `sm-breakpoint`

Reply:
its perfect!